ios/Divoc/Imported/Networking/TracingLocalPush.swift

```diff
@@ -153,12 +163,20 @@ extension TracingLocalPush: UNUserNotificationCenterDelegate {
 
   func userNotificationCenter(_: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
     guard exposureIdentifiers.contains(response.notification.request.identifier) else {
-      if(response.notification.request.identifier == notificationIdentifierCustom){
-        if let url = URL(string: "https://apps.apple.com/pt/"),
+      if (response.notification.request.identifier == notificationIdentifierCustom) {
+        if let url = URL(string: "https://apps.apple.com/pt/app/stayaway-covid/id1519479652"),
             UIApplication.shared.canOpenURL(url)
         {
           UIApplication.shared.open(url, options: [:], completionHandler: nil)
         }
+      } else if (response.notification.request.identifier == notificationIdentifierInfoBox) {
+        let url = response.notification.request.content.userInfo["url"] as? String ?? nil
+        if (url != nil) {
+          if let urlValue = URL(string: String(url!)), UIApplication.shared.canOpenURL(urlValue)
+          {
+            UIApplication.shared.open(urlValue, options: [:], completionHandler: nil)
+          }
+        }
```

Comment:
this can be done in a single if let statement, right ? and we remove the `!` from line 175

```
if
    let url = response.notification.request.content.userInfo["url"] as? String,
    let urlValue = URL(string: String(url)),
    UIApplication.shared.canOpenURL(urlValue) {

}
```

Reply:
Right, we will change before merging.